corehq/toggles/__init__.py

```diff
@@ -1567,13 +1567,6 @@ def _commtrackify(domain_name, toggle_is_enabled):
     help_link="https://confluence.dimagi.com/display/saas/Incremental+Data+Exports"
 )
 
-DISPLAY_CONDITION_ON_TABS = StaticToggle(
```

Comment:
It will be nice to cleanup the underlying 'Toggle' Couch-doc through a migration.

Reply:
(see my latest commit)